content/2017-01-13_jako-holka-v-brne.md

```diff
@@ -0,0 +1,56 @@
+Title: Jako holka v Brně jsi už napůl programátorka
+Date: 2017-01-13
+Author: Lucie Karmová
+About: Lucka pracuje jako úřednice v Brně a spravuje dotační programy pro mimořádně nadané studenty z Jihomoravského kraje. Fascinuje jí programátorská komunita v Brně, a proto se přihlásila na kurz PyLadies v Brně.
+
+**Žiju s programátorem, půlka mých přátel jsou programátoři, na počítači mi běží Fedora, mám za sebou několik neslavných programovacích začátků a nakonec jsem narazila na poklad. Mohla jsem si jen vymýšlet důvody, proč se NEučit programovat. Nakonec ale nebylo vyhnutí a přihlásila jsem se do programování v Pythonu pro holky – PyLadies.**
+**Co všechno jsem kdy nedokončila? Learn Python the Hard Way od Zeda A. Shawa, CodeAcademy nebo Programming for Everybody na Courseře. Taky jsem uvízla v polovině nulté (ano, nulté) série Korespondenčního semináře z informatiky FI MU. Celý jsem odchodila a pobrala snad jenom kurz SQLite na KISKu. Pak okolo mne prolétla registrace na PyLadies a bylo jasné, že to je to pravé. Přihlášku jsem poslala, naslibovala hory doly ohledně své ochoty makat, vzali mě a už to jelo.**
+
+
+![linux]({filename}/images/01.png)
+
+###Když v Brně hodíte kamenem, trefíte programátora
+Pro holky v Brně existuje jedinečný časoprostor. Tohle město je významné IT centrum a vývojáři se vyvažují zlatem. Zároveň jde o totálně mužské prostředí a mám pocit, že z toho samotní programátoři nejsou moc šťastní. Všichni se můžou přetrhnout, aby holkám umetli cestičku k programování. Holky v IT jsou prostě téma. 
+Dvě hodiny kurzu týdně. A nekonečně hodin domácích úkolů
+Ohromně se mi líbila systematičnost, délka i pravidelnost kurzu, krásně zpracované materiály, různé “taháky” a taky nutnost domácích úkolů. Byly zábavné, i když pro mne vlastně docela náročné. V říjnu a v listopadu jsem s nimi strávila i 10 hodin týdně. A to nepočítám hodiny manžela, který mě zachraňoval před hysterákem a počítač před vyhozením z okna. Ke konci roku naštěstí tempo zvolnilo. Beztak jsem ale od prosince stále ještě u programování hada a na asteroidy jsem ani nesáhla. Před chvílí mě manžel upozornil, že píšu článek o programování místo toho, abych _doopravdy_ programovala.
+
+
+![homework]({filename}/images/02.png)
+
+###Diskriminace! Kurz je jen pro holky
+Být holkou v libovolném věku, která se učí programovat, jsou instatní plus čtyři tečky do vaší coolness. Kdybych byla klukem, který dnes začíná v Brně s Pythonem, byl by to spíš trapas. Co že jsem sakra dělala posledních patnáct let? 
+PyLadies se snaží trochu naředit klučičí programátorské prostředí. Počítá se s tím, že ve smíšeném prostředí by se účastnice kurzu styděly ptát na věci, které přece “dávno známe ze střední”. Za mne to určitě fungovalo. Zažila jsem třeba fantastický “aha moment”, když nám Petr Viktorin vysvětloval a na tabuli kreslil, jak funguje vlnová délka. Měla jsem dojem, že mi to poprvé v životě dává smysl – a viděla jsem, že podobně to mají i ostatní. 
```

Comment:
... účastnice kurzu mohly stydět se ptát na věci, ... Pro mě to určitě fungovalo.